### PR TITLE
Fix `chainer.backend.get_array_module` documentation

### PR DESCRIPTION
**Pull request by kmaehashi**
(found during writing #4986.)

### CODE REVIEW

**Comment by toslunar** [comment]
Jenkins, test this please.

**Comment by toslunar** [comment]
Jenkins, test this please.

File: chainer/backend.py [hunk 1]
```diff
@@ -158,11 +158,11 @@ def get_array_module(*args):
 
     Args:
         args: Values to determine whether NumPy, CuPy, or ChainerX should be
-        used.
+            used.
 
     Returns:
         module: :mod:`cupy`, :mod:`numpy`, or :mod:`chainerx` is returned based
-        on the types of the arguments.
+            on the types of the arguments.
```
Comment (toslunar) [inline chainer/backend.py]: Could you revert the change to the line?

Comment (kmaehashi) [reply chainer/backend.py]: Reverted.

**Comment by toslunar** [comment]
Jenkins, test this please.
